units/en/_toctree.yml

```diff
@@ -58,3 +58,23 @@
   sections:
   - local: communication/next-units
     title: Next Units
+- title: Unit 2.2 The LlamaIndex framework
```

Comment:
@davidberenstein1957 you need to move Bonus Unit 1 after Unit 2.2 and When the next steps are published after Bonus Unit 1

So that it's:
- Unit 0
- Unit 1
- Unit 2
- Unit 2.1
- Unit 2.2
- Bonus Unit 1
- Next sections